### DISCUSSION TITLE
Using CC

### DISCUSSION THREAD

**Discussion by gigli69** [BobanSpasic/PSS-Revive#7] [Ideas]
Be able to use (assignable) CC to control the different parameters, in order to user with external controlers.

Ex: Modulator FREQ could be assigned to CC23, Carrier FREQ to CC24, etc etc

**Comment by BobanSpasic** [comment]
Hi gigli69,
I could do it, but it is a pretty big task. I don't know if it makes sense for a project with just a couple of users.
